spec/components/application_record_card/view_spec.rb

```diff
@@ -8,6 +8,7 @@ module ApplicationRecordCard
     let(:course) { provider.courses.first }
     let(:trainee) { Trainee.new(created_at: Time.zone.now, course_uuid: course.uuid, provider: provider) }
     let(:system_admin) { false }
+    let!(:academic_cycle) { create(:academic_cycle, start_date: DateTime.new(2020, 1, 1), end_date: DateTime.new(2021, 1, 1)) }
```

Comment:
It's better to avoid `let!` blocks unless you plan to reference them latter. Suggest putting the FactoryBot stuff in a `before` block. Can you also double check the other `let!` blocks see if they can also be put in a `before` block?